ToDo-Garden/ToDo-Garden/ManageGroupScene/Sources/ManageGroupScene/ManageGroupViewControllerForGuide.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  ManageGroupViewControllerForUsageGuide.swift
+//
+//
+//  Created by SONG on 8/24/24.
+//
+
+import UIKit
+
+import ToDoGardenUIComponent
+
+public final class ManageGroupViewControllerForGuide: ManageGroupViewController {
+  
+  public lazy var groupCells: [UITableViewCell] = {
+    self.groupListTableView.layoutIfNeeded()
+    let cells = self.groupListTableView.visibleCells
+    return cells
+  }()
+  
+  public override init() {
+    super.init()
+  }
+
+  public override func viewDidLoad() {
+    self.view.backgroundColor = UIColor.white
+    self.setupTableView()
+    self.setupNavigationBar()
+    self.fetchGroupList()
+  }
+  override func fetchGroupList() {
```

Comment:
`p3`
공백이 한칸 들어가도 좋을 것 같네요 ㅎㅎ
```suggestion

  override func fetchGroupList() {
```

Reply:
반영 완료 
> [54e6419](https://github.com/ToDo-Garden/ToDo-Garden/pull/442/commits/54e64199c385e839360f9315dca183ea2c088632)